tests/test_transactions.py

```diff
@@ -26,7 +26,9 @@ def emit(self, record):
         if self.pattern in record.msg:
             self.history[record.msg] += 1
 
-log = logging.getLogger("Fiona")
+
+log = logging.getLogger()
```

Comment:
Configure the root logger, not an obscure "Fiona' logger. I can't believe I was so clueless about logging for so long.